packages/components/src/sandbox/index.js

```diff
@@ -154,8 +154,17 @@ class Sandbox extends Component {
 				margin-top: 0 !important; /* Has to have !important to override inline styles. */
 				margin-bottom: 0 !important;
 			}
+			#wp-sandbox-overlay {
+				position: fixed;
+				width: 100%;
+				height: 100%;
+				top: 0;
+				left: 0;
+				right: 0;
+				bottom: 0;
+			}
 		`;
-
+		const overlayId = 'wp-sandbox-overlay';
```

Comment:
I'm not sure I see a benefit to assigning this to a variable if it's not in-fact a single source of truth (i.e. duplicated in the style assignment).

Reply:
Me neither, but the linter doesn't allow a string literal as `id` :)